orchestrate-cockroachdb-with-kubernetes.md

```diff
@@ -4,9 +4,9 @@ summary: How to orchestrate the deployment and management of an insecure 3-node
 toc: false
 ---
 
-This page shows you how to orchestrate the deployment and management of an insecure 3-node CockroachDB cluster with [Kubernetes](http://kubernetes.io/), using the alpha [PetSet](http://kubernetes.io/docs/user-guide/petset/) feature.
+This page shows you how to orchestrate the deployment and management of an insecure 3-node CockroachDB cluster with [Kubernetes](http://kubernetes.io/), using the beta [StatefulSet](http://kubernetes.io/docs/concepts/abstractions/controllers/statefulsets/) feature.
 
-{{site.data.alerts.callout_info}}Running a stateful application such as CockroachDB on Kubernetes requires using some of Kubernetes' more complex features available only in alpha versions. There are easier ways to run CockroachDB on Kubernetes for testing purposes, but the method presented here is destined to become a production deployment once Kubernetes matures sufficiently.{{site.data.alerts.end}}
+{{site.data.alerts.callout_info}}Running a stateful application such as CockroachDB on Kubernetes requires using some of Kubernetes' more complex features available only in beta versions. There are easier ways to run CockroachDB on Kubernetes for testing purposes, but the method presented here is destined to become a production deployment once Kubernetes matures sufficiently.{{site.data.alerts.end}}
```

Comment:
The "requires using some of Kubernetes' more complex features available only in beta versions" line isn't quite as correct now that petsets/statefulsets are in beta. Beta features are available in all kubernetes deployments, whereas alpha features are only available in kubernetes deployments that specifically allow them.

A better phrasing might be something more like "requires using some of Kubernetes' more complex features that are at a beta level of support". We could perhaps link to what they mean by beta - http://kubernetes.io/docs/api/#api-versioning